ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -39,6 +39,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.16-openshift-4.10
+  openstack-tests:
```

Comment:
This stanza needs to be added to other files (similar to `openstack-installer`):
* `ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml`
* `ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml`
* `ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml`
etc

Reply:
For the posterity:

```shell
buffer="$(mktemp)"
for f in $(grep -l 'openstack-installer' ci-operator/config); do
	yq --yml-output '.base_images."openstack-tests"={"name":"4.10","namespace":"ocp","tag":"openstack-tests"}' "$f" > "$buffer"
	cat < "$buffer" > "$f"
done
make update
```

Reply:
Note that we are currently using a single-branch system; the image import will have to be changed when we start using `release-4.y` branches